docs/virtual_machines/run_strategies.md

```diff
@@ -45,22 +45,22 @@ The `start`, `stop` and `restart` methods of virtctl will invoke their
 respective subresources of VirtualMachines. This can have an effect on
 the runStrategy of the VirtualMachine as below:
 
-<table>
+<table style="width: 100%">
```

Comment:
just FYI:
```
<table style="width: 100%, display: inline-table">
```
should work without adding any style to the `thead` and `tbody` elements

Reply:
Indeed! :)